README.md

```diff
@@ -104,7 +104,7 @@ Beta16 version image of ringOS
 - [ ] Book
 - [ ] Networking
 - [x] Own build tool
-- [ ] Real hardware support
+- [x] Real hardware support
```

Comment:
ringOS can't run on real hardware after Beta11, so remove this as finished.

Reply:
So you run Beta7 on real hardware.
Can I know your PC specification

Reply:
Hello

Reply:
intel i7, 8gb ram, loaded from USB

Reply:
Which version
Also, is that laptop

Reply:
Yes it is a laptop